stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -725,17 +775,19 @@
 ;; Returns (err ...) on failure.
 (define-public (stack-aggregation-commit (pox-addr { version: (buff 1), hashbytes: (buff 32) })
```

Comment:
This method should be removed. It was kept for backwards compatibility. Now, it is not compatible anymore, therefore, we only should use `stack-aggregation-commit-indexed`

Reply:
I'm curious how many apps use this function vs `-indexed`. My guess is that the easiest path forward is to remove `-indexed` but change the function's return signature here.

For example, looks like the stacks.js package uses `stack-aggregation-commit`.

Reply:
Stacks.js has not yet been updated to stacks 2.4 🙃

Either is fine to me. Jude might have a stronger opinion.

Reply:
It's fine to remove it for pox-4, but please add a changelog entry.